tests/functional/aws-node-sdk/test/object/completeMPU.js

```diff
@@ -138,5 +140,67 @@ describe('Complete MPU', () => {
                 _completeMpuAndCheckVid(uploadId, eTag, undefined, done);
             });
         });
+
+        describe('with tags set on initiation', () => {
+            const tagKey = 'keywithtags';
+
+            taggingTests.forEach(test => {
+                it(test.it, done => {
+                    const key = encodeURIComponent(test.tag.key);
+                    const value = encodeURIComponent(test.tag.value);
+                    const tagging = `${key}=${value}`;
+
+                    async.waterfall([
+                        next => s3.createMultipartUpload({
+                            Bucket: bucket,
+                            Key: tagKey,
+                            Tagging: tagging,
+                        }, (err, data) => {
+                            if (test.error) {
+                                assert(err, 'Expected err but found none');
+                                assert.strictEqual(err.code, test.error);
+                                assert.strictEqual(err.statusCode, 400);
+                                done();
+                            }
+                            next(null, data.UploadId);
+                        }),
+                        (uploadId, next) => s3.uploadPart({
+                            Bucket: bucket,
+                            Key: tagKey,
+                            PartNumber: 1,
+                            UploadId: uploadId,
+                            Body: 'foo',
+                        }, (err, data) => {
+                            next(err, data.ETag, uploadId);
+                        }),
+                        (eTag, uploadId, next) => s3.completeMultipartUpload({
+                            Bucket: bucket,
+                            Key: tagKey,
+                            UploadId: uploadId,
+                            MultipartUpload: {
+                                Parts: [{
+                                    ETag: eTag,
+                                    PartNumber: 1,
+                                }],
+                            },
+                        }, next),
+                    ], err => {
+                        assert.equal(err, null, 'Error in mpu setup: ' +
```

Comment:
You can use instead `assert.ifError(err)`, also this would allow `err` to be `undefined` which is another common case.